src/SFA.DAS.RoATPService.Application.Api/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Linq;
+using System.Reflection;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace SFA.DAS.RoATPService.Application.Api.Extensions
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static void RegisterAllTypes<T>(this IServiceCollection services, Assembly[] assemblies,
+            ServiceLifetime lifetime = ServiceLifetime.Transient)
+        {
+            var typesFromAssemblies = assemblies.SelectMany(a => a.DefinedTypes.Where(x => x.GetInterfaces().Contains(typeof(T))));
+            foreach (var type in typesFromAssemblies)
```

Comment:
I think we should apply Dave's suggestion here too....
https://github.com/SkillsFundingAgency/das-admin-service/pull/44#discussion_r318576271

Reply:
yeah, I'll do that